Source/SteveSloth/Private/MyEnemyBaseClass.cpp

```diff
@@ -9,17 +9,22 @@
  ****************************************************************************************/
 
 #include "MyEnemyBaseClass.h"
+#include "MyGenericEnemyIdleState.h"
```

Comment:
this should be in the header file

Reply:
This needs to stay here do to circular dependency. and with the include in the cpp file all im doing is grabbing a reference to it not actually using it.. In the header file i would have access to everything and i dont need that, also it causes circular dependency.

Reply:
sounds good